dbsetup.sql

```diff
@@ -11,9 +11,34 @@ create table if not exists tokens (
     expiresAt timestamp not null
 );
 
+create table if not exists artists (
+    id string primary key,
+    name string not null,
+    imageUrl string null
+);
+
+create table if not exists albums (
+    id string primary key,
+    name string not null,
+    imageUrl string not null
+);
+
+create table if not exists tracks (
+    id string primary key,
```

Comment:
I guess multiple tracks can have the same name, should there be an autogenerated ID as PK? Same goes for album and artist.